src/services/ubi/community.ts

```diff
@@ -645,6 +656,7 @@ export default class CommunityService {
             return {
                 ...manager,
                 isDeleted: !manager.user,
+                beneficiaryRegistry: manager.beneficiaryRegistry?.length,
```

Comment:
`beneficiaryRegistry` doesn't sound right. It's about, how many beneficiaries were added. Not sure what is best, as I'm not good naming things either. maybe `addedBeneficiaries`

Reply:
done!